src/Snakemake/rules/Alignment/bwa-mem.smk

```diff
@@ -67,7 +77,7 @@ rule bwa_mem:
         "logs/map/bwa/{sample}.log",
     params:
         index=config["reference"]["ref"],
-        extra=r"-R '@RG\tID:{sample}\tSM:{sample}\tPL:illumina\tPU:{sample}' -v 1",
+        extra=_bwa_params_extra,
```

Comment:
We should maybe have this as a config option:
Ex
extra=config.get("bam_extra" , r"-R '@RG\tID:{sample}\tSM:{sample}\tPL:illumina\tPU:{sample}' -v 1") alternativt   
or 
extra=r"-R '@RG\tID:{sample}\tSM:{sample}\tPL:illumina\tPU:{sample}' -v 1" + config.get("bam_extra" , "")

Could be that the second example is the best option since we in the first option will have to deal with wildcards in the config file.

Reply:
extra=r"-R '@rg\tID:{sample}\tSM:{sample}\tPL:illumina\tPU:{sample}' -v 1" + config.get("bam_extra" , "") noe implemented

Reply:
Great!